### PR TITLE
Intent to make changes

### PR DESCRIPTION
**Pull request by DG12**
This Pull Request is simply to state the changes I am finishing up.
Minimal functional changes. 
Add comments,.
Move constants to .h
Add function packet_loader to fill a test packet with interesting hexadecimal data because any utility to receive the test packets will (most  likely)  not present text.

### CODE REVIEW

File: src/integration_tests/ruuvi_interface_communication_ble_gatt_test.c [hunk 1]
```diff
@@ -16,15 +16,18 @@
...
  * @{
  */
 /**
  * @file ruuvi_interface_communication_ble_gatt_test.c
+ * @copyright Ruuvi Innovations Ltd, license BSD-3-Clause.
  * @author Otso Jousimaa <otso@ojousima.net>
  * @date 2020-06-18
- * @copyright Ruuvi Innovations Ltd, license BSD-3-Clause.
- *
+ *       2021-01-26 niksiboxi  fixup JSON
```
Comment (ojousima) [inline src/integration_tests/ruuvi_interface_communication_ble_gatt_test.c:27]: Remove these, version history is in git and changelog as appropriate

**Review by ojousima — CHANGES_REQUESTED** [review]

**Comment by ojousima** [comment]
> Add function packet_loader to fill a test packet with interesting hexadecimal data because any utility to receive the test packets will (most likely) not present text.

Sounds ok to me as long as it gets compiled only on test/debug builds, i.e. not on releases

**Comment by ojousima** [comment]
> Move constants to .h

Only if the constants are required by something else than ruuvi_interface_communication_ble_gatt_test.c. Let's keep the externally visible variables as few as possible.

**Comment by DG12** [comment]
Nearing completion.

**Pull request closed without merging**